### PR TITLE
Update Bravia TV Integration (#33774)

### PR DESCRIPTION
**Pull request by dcnielsen90**
## Proposed change

- Add documentation on properly removing Home Assistant from TV
- Minor adjustments to overall structure to improve readability

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please be aware that there is also a PR for the `current` docs: https://github.com/home-assistant/home-assistant.io/pull/13261

Maybe you can work together and combine both PRs. Another question what is the parent PR in the core repo? Otherwise this PR should go into the `current` branch.

File: source/_integrations/braviatv.markdown [hunk 1]
```diff
@@ -14,11 +14,30 @@ ha_config_flow: true
...
-If you have following configuration in your `configuration.yaml` file it will be imported to the Integrations:
+### Common Issues
+#### Previous Configurations:
+- Ensure that all entries in `configuration.yaml` have been removed and `bravia.conf` does not exist in your `.homeassistant` folder.
+- Remove entity from Home Assistant.
+- Restart Home Assistant
+- Retry [configuration](###Setup-via-the-User-Interface).
+
+#### TV does not generate new pin:
+- If you have previously set up your TV with any Home Assistant instances, you will need to remove Home Assistant from your TV. You can either factory reset your TV or follow [these instructions](https://www.sony-asia.com/electronics/support/articles/00129859). Your exact instructions may vary depending on yourmodel.
```
Comment (bieniu) [inline source/_integrations/braviatv.markdown]: Maybe you can add here something like this:
```
On the TV, go to: **Settings** -> **Network** -> **Remote device settings** -> **Deregister remote device** (menu item names may differ slightly for different TV models).
```
I checked several different TV models (2019, 2018 and 2017), and the names of the menu items were the same but in older models there are probably differences. Then factory reset may be necessary.

Comment (dcnielsen90) [reply source/_integrations/braviatv.markdown]: I agree. pushing a commit to adjust this now.

**Comment by dcnielsen90** [comment]
For clarity, I replaced my next branch with current and cherry-picked my commits. I couldn't change the branch I was pulling from and I didn't want to open a new PR.

**Review by frenck — APPROVED** [review]
Thanks, @dcnielsen90 👍

**Pull request merged**